src/App.tsx

```diff
@@ -1,14 +1,15 @@
 import React from "react";
 import "./App.css";
 import { StyleService } from "./services/StyleService";
-import {
-  Button,
-  NumericInput,
-} from "@blueprintjs/core";
-import {FallingStarsView} from "./FallingStarsView";
+import { Button, NumericInput } from "@blueprintjs/core";
+import { FallingStarsView } from "./FallingStarsView";
 
-export class App extends React.Component<{}, {}> {
+import { ProfilePage } from "./Profile";
+import { LeaderBoardPage } from "./Leaderboard";
+
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
 
+export class StarPage extends React.Component<{}, {}> {
```

Comment:
i think this should be a new separate file so theres one component per file

Reply:
moved all comps into their own files and added a Pages and Components folder